File: lab19/task2.js
function checkNumber(num) {
    if (num > 0) {
        console.log("The number is positive");
    } else if (num < 0) {
        console.log("The number is negative");
    } else {
        console.log("The number is zero");
    }
}

// Test the function with different numbers
checkNumber(-5); // Output: The number is negative
checkNumber(0);  // Output: The number is zero
checkNumber(7);  // Output: The number is positive

File: lab19/task2.py
def check_number(num):
    if num > 0:
        print("The number is positive")
    elif num < 0:
        print("The number is negative")
    else:
        print("The number is zero")

# Test the function with different numbers
check_number(-5)  # Output: The number is negative
check_number(0)   # Output: The number is zero
check_number(7)   # Output: The number is positive
